### ISSUE TITLE
Too long longUrl

### ISSUE THREAD

**Issue by Celforyon** [shlinkio/shlink#507]
#### How Shlink is set-up

* Shlink Version: latest (docker shlinkio/shlink)
* PHP Version: 7.3.8
* How do you serve Shlink: Docker image
* Database engine used: MariaDB (Ver 15.1 Distrib 10.4.8-MariaDB)

#### Summary

Error: `Shlink.ERROR - Unexpected error creating short url.`
Seems to happen when given a too long longUrl
```
PDOException: SQLSTATE[22001]: String data, right truncated: 1406 Data too long for column 'original_url' at row 1 in /etc/shlink/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOStatement.php:117
```

#### Current behavior

Short URL link not generated/provided.

#### Expected behavior

Short URL link generated and provided, even for long longUrl.

#### How to reproduce

```sh
curl -X POST "https://${DOMAIN}/rest/v1/short-urls" -H "X-Api-Key: ${KEY}" -H "Content-Type: application/json" -d "{\"longUrl\":\"${URL}\"}"
```
where `DOMAIN` and `KEY` correspond to a valid shlink server and `URL` is too long.
example of value for `URL`:
```
https://godbolt.org/#z:OYLghAFBqd5QCxAYwPYBMCmBRdBLAF1QCcAaPECAKxAEZSAbAQwDtRkBSAJgCFufSAZ1QBXYskwgA5NwDMeFsgYisAag6yAwnlSCCxTEwC2G7BwAMAQTkKlKzOq0A3TMiLFTF613mLlajU1kPXQGPAAjTysbP3tHTT1iBWAo7187AK0QzAAPCQAHAlSvEUFk1RZjTEF8pglVEI0%2BaKsCTCN85jb4pSZBQVUAFVTe/tUAZQI6gGt1AHYedStVFfykpyY2kCXLFb3VFzcSQOHZbFVMBnbBJv2AejuLq6NMFgIBry89/JFwsORtl99gdUHh0KofoIEBBBqo0Cw9NwAGwASluqgeENKCCe7VeBB2wJBYIhqHyEDRskWRJpGMe%2BTJuJeb0JwNhRHJKLhqARRSptPujwMBDELFUHKZ%2BNZ%2ByB%2B3CqFQDAuHQIAE8KdzefNqYLVMLReLiCIHHgAGbKwqqgB0sqJ%2BuIYsu1yt7UtFNutpWHDmABEdt6fR6WpY2h0ug5AqMBqczFYnKDwZMZidTCAQJCcTDNYiuEimVzvYtZZimPl8q9wWh8qrVKhzbV%2BphwU7mQTZS3BFaMwB9cIzCAtyk6/2%2B/2tV3hnrMMYxrzxklJ5DTFNnNMM/KqDWF6Vmzcdl0q9UogsLaV7AgIYioADutZEBG7de7xFYwEwEG4XEXy60qfTZIpbZXTVBopiXT8h2lANZWLIV2lQFxVGnAkWyldtnk7dde37SDaU%2BUdPnHMNNgjLQoyGVJYW/FdsDTCUNXhPRtR3c0Bwwg83WPZjPXFS8bzvB8nxfNh30/ajf1XEAOUAi0QL0ZMuC4SCYN9GDlhWTF7TFKsazrJC%2BhQ540PUvVMBFB0mU7Psl3dflgXwv1CMsBQCSMJgFC3U9ZX0GtCx48TtDeUwBRc8TqUxeSl1rc0XI%2BEy9gCxJkmCpK2DC3VItmPTUuAOLrHijTHjclg8B%2BScXNAmYzxWUKwOmLtsQgOZlIK7l73iQJVFqmYrWkgsJM0CsGCDfLdmBTFitKkRJxyyqop4nLvwaqEP0UhBLgYVAIJGok0HawJOsWureoA/rNEOgh0DTIaduBI6evXWyi1a%2B6ly7U7bvmP1kE2ZBM1yAoCB0bSeRzPNchPZ6xv2CRiGIDqtHURTsDyTBCmBwFFIRzQLhyK1rwQTYtwGm7%2BR4rTVAAWloJoVMc6JRykFFGGkABWKRSBYaRzA51BpHO3h%2BAaURxAjHxaA5ghuaZ5nphAVnzBZqQABYOa5qQedIPmpA5wQQEVqWNaZ0g4FgJA0A6PArjICgIAt/IrcwYgQGAABOLhSFNK22mIPWIHCaXSHCBQmGIVVpAl0gLdbAB5FgGHDo3SCwYrgCuQP8AMNw8BcPWk9yVx70kKRI5cy5A7CcIXzDzQMGLyP9DwIwI%2BN5g2BQfh%2BEYCI9cgZmySB0HpF1kWJDoZmGDZtXA%2B1nIAA4kUppFldUYBkGQVRXatLhN1wQgSCR2R6FUWvLetw/aC5AW%2BF4SXpePUg5YVpXVc5mfh6EfXSENnmH8nqQuDTyTtrO%2BRsH4uF9hjZWQA%3D
```

**Comment by acelaya** [comment]
I have increased the size of that field several times due to this issue. I think I will have to find a more definitive solution.

I will at least increase the size again.

Thanks for reporting :)

**Comment by acelaya** [comment]
I have increased the size of this columng from `1024` to `2048` characters, which seems to be the theoretical maximum value according to the HTTP standard: https://stackoverflow.com/a/417184

The URL that failed for you is ~1600 characters long, so it should work with the new size.

The fix will be part of the next release (`v1.20.0`).
